From ddd1a8c5bd6d59191d40672221972b675e3eaf99 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Wed, 25 Mar 2015 08:44:17 -0700
Subject: [PATCH 4/9] Remove Qt4 stuff from the documentation

---
 README                    | 28 +++++-----------------------
 examples/use-qtxdg/README | 11 +++--------
 2 files changed, 8 insertions(+), 31 deletions(-)

diff --git a/README b/README
index 9302686..4abf265 100644
--- a/README
+++ b/README
@@ -1,20 +1,11 @@
 Overview
 ========
 libqtxdg is An Qt implementation of freedesktop.org xdg specifications.
-It can be built with Qt4 and Qt5
+It's built with Qt5.
 
 
 Dependencies
 ============
-Qt4 build:
-    Qt4
-    libmagic OR QtMimeTypes
-
-QtMimeTypes the preffered. libmagic is deprecated and may be dropped in future
-releases.
-QtMimeTypes can be found at: https://qt.gitorious.org/qtplayground/mimetypes
-
-Qt5 build:
     Qt5
 
 
@@ -24,20 +15,11 @@ libqtxdg uses the CMake build system. Everything that applies to CMake also
 applies here.
 
 Configuration options:
-    USE_QT5     	Builds with Qt5, defaults to then environment variable
-                        with the same name.
-
-    USE_QTMIMETYPES	It only affects the Qt4 build. Builds using QtMimeTypes.
-			Defaults to On. If set to OFF libmagic will be used.
-
     BUILD_TESTS         Builds tests, defaults to OFF
 
 Configuration Examples:
-    Build with Qt5 and build self tests:
-        cmake -DUSE_QT5=ON -DBUILD_TESTS=ON ..
-
-    Build with Qt4 and no tests using QtMimeTypes
-        cmake -DUSE_QT5=OFF ..
+    Build library  and build self tests:
+        cmake -DBUILD_TESTS=ON ..
 
-    Build with Qt4, no tests and using libmagic
-        cmake -DUSE_QT5=OFF -DUSE_QTMIMETYPES=OFF ..
+    Build the library without building self tests
+        cmake ..
diff --git a/examples/use-qtxdg/README b/examples/use-qtxdg/README
index 9f9d687..dada0aa 100644
--- a/examples/use-qtxdg/README
+++ b/examples/use-qtxdg/README
@@ -1,9 +1,4 @@
-An example of how to write an CMakeLists.txt to use libqtxdg in a portable
-way.
+An example of how to write an CMakeLists.txt to use libqtxdg.
 
-Building with Qt4 is the default. Ex:
-cmake..
-
-To build with Qt5 just pass to set the USE_QT5 to Yes. Ex:
-
-cmake -DUSE_QT5=On ..
+To build this example just use:
+    cmake ..
-- 
1.9.0

