From 04630a0f1f7ccee3f3eac11ed6abb3a75c7595a0 Mon Sep 17 00:00:00 2001
From: Helio Chissini de Castro <helio@kde.org>
Date: Sun, 15 Feb 2015 11:18:06 -0200
Subject: [PATCH 14/15] - Fix compilation when using internal mime

Signed-off-by: Helio Chissini de Castro <helio@kde.org>
---
 xdgmime.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xdgmime.h b/xdgmime.h
index 9cc4717..7c4a201 100644
--- a/xdgmime.h
+++ b/xdgmime.h
@@ -39,7 +39,7 @@ struct XdgMimeData;
 
 /*! @brief The XdgMimeInfo class provides mime information about file.
  */
-class QTXDG_DEPRECATED QTXQD_API XdgMimeInfo
+class QTXDG_DEPRECATED QTXDG_API XdgMimeInfo
 {
 public:
     /// Constructs a XdgMimeInfo with the mimeType type.
-- 
1.9.0

