From 288d118200e772916e8cd901a67e8cfe236a7ab9 Mon Sep 17 00:00:00 2001
From: Paulo Lieuthier <paulolieuthier@gmail.com>
Date: Thu, 5 Mar 2015 11:40:36 -0300
Subject: [PATCH 20/27] XdgDesktopFileCache: Retrieve apps of a category

---
 xdgdesktopfile.cpp | 14 ++++++++++++++
 xdgdesktopfile.h   |  8 ++++++++
 2 files changed, 22 insertions(+)

diff --git a/xdgdesktopfile.cpp b/xdgdesktopfile.cpp
index a3d7bf2..7cdd5dd 100644
--- a/xdgdesktopfile.cpp
+++ b/xdgdesktopfile.cpp
@@ -57,6 +57,7 @@
 #include <QProcess>
 #include <QUrl>
 #include <QDesktopServices>
+#include <QStringBuilder> // for the % operator
 #if QT_VERSION < QT_VERSION_CHECK(5,0,0)
 #else
 #include <QStandardPaths>
@@ -1584,6 +1585,19 @@ void XdgDesktopFileCache::initialize()
     }
 }
 
+QList<XdgDesktopFile*> XdgDesktopFileCache::getAppsOfCategory(const QString& category)
+{
+    QList<XdgDesktopFile*> list;
+    const QString _category = category.toUpper();
+    foreach (XdgDesktopFile *desktopFile, instance().m_fileCache.values())
+    {
+        QStringList categories = desktopFile->value("Categories").toString().toUpper().split(QLatin1Char(';'));
+        if (!categories.isEmpty() && (categories.contains(_category) || categories.contains(QLatin1String("X-") % _category)))
+            list.append(desktopFile);
+    }
+    return list;
+}
+
 QList<XdgDesktopFile*>  XdgDesktopFileCache::getApps(const QString& mimetype)
 {
     return instance().m_defaultAppsCache.value(mimetype);
diff --git a/xdgdesktopfile.h b/xdgdesktopfile.h
index 2117ec0..b71628e 100644
--- a/xdgdesktopfile.h
+++ b/xdgdesktopfile.h
@@ -231,6 +231,14 @@ public:
     static XdgDesktopFile* getDefaultApp(const QString& mimeType);
     static QSettings::Format desktopFileSettingsFormat();
 
+    /*! Return all desktop apps that have category for their Categories key
+     * Note that, according to xdg's spec, for non-standard categories "X-"
+     * is added to the beginning of the category's name. This method takes care
+     * of both cases.
+     * See http://standards.freedesktop.org/menu-spec/menu-spec-latest.html#desktop-entry-extensions
+     */
+    static QList<XdgDesktopFile*> getAppsOfCategory(const QString &category);
+
 private:
     static XdgDesktopFileCache & instance();
     static XdgDesktopFile* load(const QString & fileName);
-- 
1.9.0

