From 641754a77b9692c3721128cde415ef13c770f88c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Tue, 10 Mar 2015 09:21:09 -0700
Subject: [PATCH 27/27] XdgDirs: Updates documentation.

---
 xdgdirs.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/xdgdirs.h b/xdgdirs.h
index ea7d4f1..0ac1298 100644
--- a/xdgdirs.h
+++ b/xdgdirs.h
@@ -38,6 +38,8 @@
  * This specification defines where these files should be looked for by defining one or more base
  * directories relative to which files should be located.
  *
+ * All postfix parameters should start with an '/' slash.
+ *
  * @sa http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
  */
 
@@ -91,7 +93,7 @@ public:
     /*! @brief Returns a list of all directories that corresponds to the $XDG_DATA_DIRS.
      * $XDG_DATA_DIRS defines the preference-ordered set of base directories to search for data
      * files in addition to the $XDG_DATA_HOME base directory. If $XDG_DATA_DIRS is either not set
-     * or empty, a value equal to /usr/local/share/:/usr/share/ should be used.
+     * or empty, a value equal to /usr/local/share:/usr/share is used.
      *
      * If the postfix is not empty it will append to end of each returned directory.
      */
-- 
1.9.0

