diff -up libqtxdg-3.8.0/src/qtxdg/xdgdesktopfile.cpp.1~ libqtxdg-3.8.0/src/qtxdg/xdgdesktopfile.cpp
--- libqtxdg-3.8.0/src/qtxdg/xdgdesktopfile.cpp.1~	2021-11-05 18:59:19.904633563 +0100
+++ libqtxdg-3.8.0/src/qtxdg/xdgdesktopfile.cpp	2021-11-05 19:00:47.722769314 +0100
@@ -478,7 +478,7 @@ bool XdgDesktopFileData::startApplicatio
         else
         {
             qWarning() << "XdgDesktopFileData::startApplicationDetached(): Using fallback terminal (xterm).";
-            terminalCommand = QStringLiteral("xterm");
+            terminalCommand = QStringLiteral("xvt");
         }
 
         delete terminal;
