diff -Naur libqtxdg-3.3.0/src/qtxdg/xdgdesktopfile.cpp libqtxdg-3.3.0.tpg/src/qtxdg/xdgdesktopfile.cpp
--- libqtxdg-3.3.0/src/qtxdg/xdgdesktopfile.cpp	2016-09-17 12:06:08.000000000 +0000
+++ libqtxdg-3.3.0.tpg/src/qtxdg/xdgdesktopfile.cpp	2016-10-06 21:04:18.855369676 +0000
@@ -402,7 +402,7 @@
     {
         QString term = QString::fromLocal8Bit(qgetenv("TERM"));
         if (term.isEmpty())
-            term = QLatin1String("xterm");
+            term = QLatin1String("xvt");
 
         args.prepend(QLatin1String("-e"));
         args.prepend(term);
